api/src/pcapi/routes/backoffice/templates/components/links.html

```diff
@@ -1,11 +1,18 @@
 {# ----- internal links (backoffice) ----- #}
-{% macro build_public_user_name_to_details_link(user) %}
+{% macro build_public_user_name_and_id_to_details_link(user) %}
```

Comment:
Par homogénéité tu pouvais utiliser la fonction existante. Certes l’id sera affiché mais au moins ce sera identiques aux autres pages sur lesquelles un tel lien apparaît.

Reply:
J'avais renommé justement pour éviter d'afficher l'id.
Ok j'utilise le même alors.

Reply:
par contre, vue que je vais effacer ma macro, je remet le nom d'avant concernant cette macro? 
build_public_user_name_to_details_link ? ou je garde le and_id?

Reply:
Tu remets le nom original, pour ne pas avoir de changement de l'existant.